src/Sylius/Bundle/CoreBundle/Resources/config/services.xml

```diff
@@ -150,6 +150,12 @@
             <argument type="service" id="sylius.variant_resolver.default" />
         </service>
 
+        <service id="sylius.order_processing.order_exchange_rate_and_currency_updater" class="Sylius\Component\Core\OrderProcessing\OrderExchangeRateAndCurrencyUpdater">
```

Comment:
It wasn't consistent - [check em](https://github.com/Sylius/Sylius/pull/5847/files#diff-8a0b8124c71d8dcd79c6a62575307bacR521)